src/index.html

```diff
@@ -58,7 +58,7 @@
 
     <main class="row">
       <section class="girlscode fullwidth col-12" data-parallax="scroll" data-image-src="./img/background.jpg">
-        <img src="/img/girlscodelincoln.png" alt="Girls Code Lincoln" />
+        <img src="/img/girlscodelincoln.svg" alt="Girls Code Lincoln" />
```

Comment:
Why did you change all the URLs relative to root (/img) instead of (img)? It broke images when browsing locally.

Reply:
I don't have a great reason, I was just trying to standardize it. I can change them all back to relative if you'd like.

By the way, I've been running the source folder on local host (refer to README) to make sure changes look exactly as they would when deployed on local machines